all_test.go

```diff
@@ -26,6 +26,7 @@ import (
 	"time"
 
 	"github.com/cznic/strutil"
+	"github.com/kr/pretty"
```

Comment:
Let's please not introduce a new dependency. This package is part of some Linux distributions and all the respective package maintainers would have to take care of it.